native/chat/chat-thread-background.react.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import ChatThreadList from './chat-thread-list.react';
+import type { ChatTopTabsNavigationProp } from './chat.react';
+import { threadInBackgroundChatList } from 'lib/shared/thread-utils';
+import { type ThreadInfo, type RawThreadInfo } from 'lib/types/thread-types';
+
+type BackgroundChatThreadListProps = {
+  navigation: ChatTopTabsNavigationProp<'BackgroundChatThreadList'>,
+};
+export default function BackgroundChatThreadList(
+  props: BackgroundChatThreadListProps,
+) {
+  const filterFunction = React.useCallback(
+    (item: ?(ThreadInfo | RawThreadInfo)) => {
```

Comment:
It looks like you've defined a useless function here. Just pass `threadInBackgroundChatList` in directly, it's the same thing

Reply:
Actually, we don't even need `useCallback` here since the function definition never changes. Sorry for suggesting that, it was my bad. We can just pass `threadInBackgroundChatList` into `ChatThreadList` directly